src/components/PlantCareBody/PlantCareBody.tsx

```diff
@@ -0,0 +1,28 @@
+import { View } from 'react-native';
+import React, { SetStateAction, useCallback, useState } from 'react';
+import { useStyles } from 'react-native-unistyles';
+
+import { PlantingStep } from '@src/api/model';
+import { PlantingSteps } from '../PlantingSteps/PlantingSteps';
+import { PlantingStepTypes } from '../PlantingStepTypes/PlantingStepTypes';
+import { styleSheet } from './PlantCareBody.style';
+import { PlantingStepDays } from '../PlantingStepDays/PlantingStepDays';
+
+export type TStapDay = SetStateAction<PlantingStep | null>;
+
+export const PlantCareBody = () => {
+	const { styles } = useStyles(styleSheet);
+	const [curStepDay, setCurStepDay] = useState<PlantingStep | null>(null);
+
+	const handlerCurStepDay = useCallback((day: TStapDay) => {
+		setCurStepDay(day);
+	}, []);
```

Comment:
redundant callback, you can pass `setCurStepDay` directly as props

Reply:
done